app/controllers/concerns/health_careplan.rb

```diff
@@ -62,7 +62,7 @@ def careplan_pdf_coversheet
         wait_until: 'networkidle0',
         print_background: true,
       }
-      CombinePDF.parse(Grover.new(wrap_in_html(coversheet), **grover_options).to_pdf, allow_optional_content: true)
+      Pdfunite.join(Grover.new(wrap_in_html(coversheet), **grover_options).to_pdf) { |obj| obj }
```

Comment:
I guess the pdf unite gem's block is required? Seems like an awkward API. Maybe we provide a wrapper?
```ruby
class PdfGenerator
  # provides a default block
  def self.merge_pdfs(objects, &block)
    Pdfunite.join(objects, &(block || proc { |pdf| pdf }))
  end
end
```

```suggestion
      PdfGenerator.merge_pdfs(Grover.new(wrap_in_html(coversheet), **grover_options).to_pdf)
```

It would be nice to tidy our Pdf rendering up into one class so we don't have to call grover directly either, but that seems out of scope

Reply:
Yeah, that one tripped me up, ended up digging into the source.  Moving it to the PdfGenerator is a great idea, thanks!

Reply:
@ttoomey I went with a slightly different approach to try and enforce some consistency in usage (which we actually have.)  Do you think this is ok, or do you prefer your more flexible approach?